packages/core/tests/test_skill_manager.py

```
@@ -0,0 +1,301 @@
+        """Test running a non-existent skill."""
+        result = mock_skill_manager.run_skill("nonexistent", {})
+        assert "error" in result
+    
+    def test_run_skill_with_args(self, mock_skill_manager, sample_skill_dir):
+        """Test running a skill with arguments."""
+        args = {"param1": "value1", "param2": 42}
+        result = mock_skill_manager.run_skill("test-skill", args)
+        assert "error" not in result
+    
+    @patch("subprocess.run")
+    def test_run_skill_timeout(self, mock_run, mock_skill_manager, sample_skill_dir):
```

Ruff (0.14.10)

Unused method argument: `sample_skill_dir`

(ARG002)